Vision/src/Annotation/SafeSearch.php

```diff
@@ -191,4 +201,25 @@ public function isViolent($strength = self::STRENGTH_LOW)
     {
         return $this->likelihood($this->info['violence'], $strength);
     }
+
+    /**
+     * Check whether the image contains racy content
+     *
+     * Example:
+     * ```
+     * if ($safeSearch->isRacy()) {
+     *     echo "Image contains racy content.";
+     * }
+     * ```
+     *
+     * @param  string $strength [optional] Value should be one of "low",
+     *         "medium" or "high". Recommended usage is via `Face::STRENGTH_*`
```

Comment:
This documentation is a bit off. It's my fault, I apologize. Can you change `Face::STRENGTH_*` to `SafeSearch::STRENGTH_*`? (If you don't mind applying that change across the entire file, I'd appreciate it. :) )